src/Book.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Objects;
+
+public class Book {
+
+    private final String title;
+    private String Author;
+    private int publishingYear;
+
+    public Book(String title, String Author, int publishingYear) {
```

Comment:
```suggestion
    public Book(String title, String author, int publishingYear) {
```
названия полей, переменных, методов нужно писать с маленькой буквы